### PR TITLE
Adding support for optional uid.

### PR DESCRIPTION
**Pull request by galp**
Hi @rnelson0, 

I have added optional support for specifying a uid for the user. Not sure if you want to add this but I though I will offer

Cheers

A.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -29,11 +29,13 @@
   $comment,
   $groups,
   $password,
+  $uid                 = undef,
   $last_change         = 0,
   $password_max_age    = 90,
   $shell               = '/bin/bash',
   $home                = "/home/${name}",
   $ssh_authorized_keys = [],
+
```
Comment (rnelson0) [inline manifests/init.pp]: Can you remove this blank line?

File: manifests/init.pp [hunk 2]
```diff
@@ -29,11 +29,13 @@
   $comment,
   $groups,
   $password,
+  $uid                 = undef,
```
Comment (rnelson0) [inline manifests/init.pp]: Can you add a comment above around line 13 with an example?

**Comment by rnelson0** [comment]
@galp Thank you so much for this contribution! It looks great and I'd like to merge it. I added two comments, if you can address those I'll merge it right away and push a new minor version to the forge in the evening.

**Comment by rnelson0** [comment]
Thank you @galp!

**Comment by galp** [comment]
Thank you for the great work!

**Comment by rnelson0** [comment]
@galp v1.0.3 is now available from the forge

**Pull request merged**